tasks/install_waf.yml

```diff
@@ -44,13 +44,34 @@
     force: true
   changed_when: "'molecule-idempotence-notest' not in ansible_skip_tags"
 
-- name: Copy crs-setup.conf.example to crs-setup.conf
+- name: Check if default config is used
+  ansible.builtin.set_fact:
+    crs_default_config_used: "{{ crs_conf_src_template.endswith('/crs-setup.conf.example') }}"
+
+# Derive a default config from the example config. This is only done once if no custom config is used.
+- name: Copy crs-setup.conf.example to crs-setup.conf.default
   ansible.builtin.copy:
     src: "{{ crs_dir }}/crs-setup.conf.example"
-    dest: "{{ crs_dir }}/crs-setup.conf"
+    dest: "{{ crs_dir }}/crs-setup.conf.default"
     mode: '0644'
     remote_src: true
     force: false
+  when: crs_default_config_used
+
+- name: Patch crs-setup.conf.default
+  ansible.builtin.blockinfile:
+    path: "{{ crs_dir }}/crs-setup.conf.default"
+    marker: "# {mark} ANSIBLE MANAGED BLOCK"
+    insertafter: "#   setvar:tx.paranoia_level=1"
+    block: |-
```

Comment:
Maybe adding a comment would help other users to understand why we have to patch it